readme.md

```diff
@@ -0,0 +1,63 @@
+# RecsByText
+
+Recommend movies to your friends and ask them for recommendations all through a convenient SMS interface.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes. See deployment for notes on how to deploy the project on a live system.
+
+### Prerequisites
+
+You should have the following installed on your local machine:
+
+- Python 3.X with pip
+- Postgres 11.X
+- [ngrok](https://ngrok.com/)
+
+This app uses [Twilio](https://www.twilio.com/) to send and receive messages, so you will need an account, phone number, and API key.
+
+### Running Locally
+
+- Make sure Postgres is running and a database is created using the credentials in `./moviesByPhone/settings.py`.
+- Clone this repository and navigate to the directory:
+
+```bash
+git clone karllhughes/movies-by-phone
+cd movies-by-phone
+```
+
```

Comment:
Might want to suggest creating a virtual environment here, `python -m venv venv`, before install the dependancies.

Reply:
Good call, added.